styles.css.scss

```diff
@@ -0,0 +1,175 @@
+// RESET, MIXINS AND VARIABLES
+
+@import 'reset';
+
+@mixin borders-and-margins {
+  border: $standard-border;
+  box-sizing: border-box;
+  margin: 5px;
+}
+
+$standard-border: 2px black solid;
+$nav-bar-height: 45px;
+
+
+
+// NAV BAR
+
+body {
+  @media (max-width: 990px) {
+    padding-top: $nav-bar-height;
+  }
+  padding-top: $nav-bar-height + 5px;
+}
+
+nav {
+  position: fixed;
+  top: 0;
+  z-index: 1;
+
+  box-sizing: border-box;
```

Comment:
Okay it looks like the tan was being used for your <nav>; I wasn't sure if the mockup wanted a full box border or just a border-bottom.